README.md

```diff
@@ -75,17 +75,24 @@ How many riders represent team Ducati?
 
 ## Building
 
-### Linux Ubuntu 16.04
+### Compiling
 
 Requirements:
 
 * The RedisGraph repository: `git clone https://github.com/RedisLabsModules/RedisGraph.git`
-* The build-essential and cmake packages: `apt-get install build-essential cmake`
+* Python 2.7
 
-To build, run `make` in the project's directory
+To build, run ```./setup-build.py``` followed by `make` in the project's directory
```

Comment:
To build there shouldn't be any requirement for python,
for testing, now that's a different story...
Let's keep building this project to a simple call to Make.

Reply:
The setup script for the build requirements is voluntary. It may as well serve as documentation. The reason we need python there is because it allows us to easily determine the platform we run on. The user may prefer to install packages manually.